src/cmd/linuxkit/pkglib/manifest_push_script.go

```diff
@@ -0,0 +1,112 @@
+package pkglib
```

Comment:
This file is auto-generated using, right? I have never used the gogen stuff, but wouldn't expect that this file needs to be checked in...

Reply:
Yeah, better to check in, Go expects that. Otherwise you need to add a Makefile to generate first.

Reply:
This is just short term. Once all other uses of `packaging.mk` are removed I can move `scripts/push-manifest.sh` into this source dir, which will result in it being in the docker build context used for the building the binary and I can switch to using `go gen` at build time.